applications/StructuralMechanicsApplication/custom_elements/prestress_membrane_element.cpp

```diff
@@ -134,6 +134,8 @@ void PrestressMembraneElement::Initialize()
     mThickness0 = GetProperties()[THICKNESS];
```

Comment:
but my concern is, if in the later development (for example optimization), in case the thickness is changeing during the steps, the THICKNESS in properties will also change, am I right?